machine/drivers/azure.md

```diff
@@ -35,6 +37,29 @@ After authenticating, the driver remembers your credentials up to two weeks.
 
 [aad-docs]: https://azure.microsoft.com/documentation/articles/virtual-machines-windows-create-aad-work-id/
 
+## Authentication (service principal account)
+1. Create an Azure Service Principal account ([described here][sp-1] or [here][sp-2]) to get a
```

Comment:
The link to the Microsoft docs could be added directly in the markdown link. 
The link to the Packer docs is not needed.